### PR TITLE
Add verbose option for plugins

### LINKED ISSUE #100: Debug option at plugins
Just an idea:

Currently the plugins have 4 lines of console log each if everything goes without any errors:
```
[SearchIndex] Compiling file public/wp-search.json
[SearchIndex] Successfully fetched posts from http://xxxxxxxxx/graphql
[SearchIndex] Created directory ./public
[SearchIndex] Successfully wrote file to public/wp-search.json
```
Making a total of 12 lines if all plugins are passed. I was thinking that maybe is a good idea to just log 1 line (per plugin) if everthing goes as supossed, something like: `Successfully created: wp-search.json`. 

In order to not lose all the useful console logs, we can grab a new boolean variable debug at each plugin:

`const { env, outputDirectory, outputName, debug = false } = nextConfig;`

And set all console logs with an && operator based on that debug variable. 

`debug && console.log('....')`

Like so, by default only one line is logged (a new final log added after plugin.postcreate, not conditional). The errors will still work as they are handle as `throw new Error`. Also in case is needed, we can pass the variable **debug = true** at `next.config.js` and activate all the logs accross the code.

shot

What do you think?

### PR DESCRIPTION
**Pull request by GuilleAngulo**
### Description
Updates plugin message log: By default, without errors, one line per plugin will print reporting successful operation. It is possible to pass a boolean variable, `verbose`, at _next.config.js_ set to **true** to print previous informational messages.

Also errors are now handled regardless of message logging. Errors wont block the process as they are handled by try-catch.

One extra function is added to print colors at console, making easier to understand info messages from errors.

## Screenshots

|Info                    |Errors
| ---------------------------------- | ---------------------------------- |
| shot | error |

Fix #100

### CODE REVIEW

File: next.config.js [hunk 1]
```diff
@@ -12,6 +12,8 @@ module.exports = withPlugins([[indexSearch], [feed], [sitemap]], {
 
   trailingSlash: true,
 
+  //verbose: true,
```
Comment (colbyfayock) [inline next.config.js:15]: would you mind adding a quick comment above this line for someone hopping in that might not necessarily be familiar with the verbose concept?

**Comment by colbyfayock** [comment]
this looks great @GuilleAngulo - thinking a quick comment above the verbose option would help people use it then we can get this merged in

File: next.config.js (hunk 1 above)
Comment (GuilleAngulo) [reply next.config.js:15]: Sure thing! Changing it

Comment (colbyfayock) [reply next.config.js:17]: perfect thanks

**Pull request merged**